src/include/packager/repositories_include.rb

```diff
@@ -340,9 +341,17 @@ def autorefresh_for?(url)
 
     # scan the repository URL and return the available products
     # @return [Array<Y2Packager::ProductLocation>] Found products
-    def scan_products(expanded_url, original_url)
-      new_repos = Pkg.RepositoryScan(expanded_url)
-      found_products = new_repos.map { |(name, dir)| Y2Packager::ProductLocation.new(name, dir) }
+    def scan_products(_expanded_url, original_url)
+      # use the selected base product during installation,
+      # in installed system use the installed base product
+      base_product = if Stage.initial
+        Y2Packager::Product.selected_base.name
+      else
+        Y2Packager::Product.installed_base_product.name
```

Comment:
when I see this API, it looks inconsistent, why it is selected_base and installed_base_product? both should be base_product or both only base.

Reply:
Yes, I also noticed that, but that's out of the scope of this PR...